crates/ibc/src/mock/header.rs

```diff
@@ -71,9 +71,18 @@ impl MockHeader {
     }
 
     pub fn new(height: Height) -> Self {
-        Self {
-            height,
-            timestamp: Timestamp::now(),
+        cfg_if::cfg_if! {
+            if #[cfg(any(test, feature = "mocks"))]  {
```

Comment:
```suggestion
            if #[cfg(any(test, feature = "std"))]  {
```
This file already only gets compiled only if `any(test, feature = "mocks", feature = "mocks-no-std")`. This better represents when we want to use `Timestamp::now()` vs `Timestamp::none()`